packages/flutter/test/material/chip_test.dart

```diff
@@ -923,6 +1007,200 @@ void main() {
     expect(find.byKey(deleteButtonKey), findsNothing);
   }, skip: isBrowser);
 
+  testWidgets('Chip creates centered, unique ripple when label is tapped', (WidgetTester tester) async {
+    // Creates a chip with a delete button.
+    final UniqueKey labelKey = UniqueKey();
+    final UniqueKey deleteButtonKey = UniqueKey();
+
+    await tester.pumpWidget(
+      _chipWithOptionalDeleteButton(
+        labelKey: labelKey,
+        deleteButtonKey: deleteButtonKey,
+        deletable: true,
+      ),
+    );
+
+    final RenderBox box = Material.of(tester.element(find.byType(InkWell))) as dynamic;
```

Comment:
why `as dynamic` and not `as Renderbox`? Also, is there not a method built into `tester` to give you the renderbox?

Reply:
I'm actually really surprised this doesn't crash at runtime: `Material.of` returns a `MaterialInkController`, which is not a `RenderBox` or `RenderBox` subclass (it is a base class).

If what you want is the `RenderBox` of the material used by the `InkWell`, you can use:
```dart
final RenderBox box = tester.renderObject(find.byType(PhysicalShape));
```
There may be more than one on the `Chip`, so you'll have to figure out which one you're looking for.

Reply:
Actually, looks like there is already a finder for the material in the `chip_test.dart`, called `getMaterialBox`.  Is that what you wanted?

Reply:
`getMaterialBox` works here. It looks for the first `RenderBox` with a `RawChip` as its descendant, which should be the same object.